src/i18n/en-US/index.ts

```diff
@@ -1035,8 +1036,12 @@ export default {
       text: '3rd Party Bridge. Fast and reliable crypto transfers across networks to Astar zkEVM.',
     },
     warning32blocks: 'It could take around 10~20mins or more to finalize',
+    warningLzWithdrawal:
+      'It could take more than 7 days to finalize the bridge transaction from Astar zkEVM to Astar EVM due to block confirmation fixed by LayerZero',
```

Comment:
Time to bridge is reduced to 6 hours

Reply:
Fixed, btw, I'm still waiting for DOT to reduce to 6 hours.